src/Dapper.AOT/UseColumnAttributeAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Dapper
+{
+    /// <summary>
+    /// Specifies whether to use [System.ComponentModel.DataAnnotations.Schema.ColumnAttribute] for additional behavioral configuration
+    /// </summary>
+    public class UseColumnAttributeAttribute : Attribute
```

Comment:
what is the allowed usage here? can it be applied at the module level, or is it only respected when attached to the same type? are more than one allowed? can we add `[AttributeUsage(...)]` here accordingly

Reply:
added such an expression.
`[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false)]`

let me know if such `AttributeUsage` is correct